### PR TITLE
Issue 19802 add descriptions and link to starter page

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

**Review by hmoreras — APPROVED** [review]

File: src/app/portlets/dot-starter/dot-starter.component.html [hunk 1]
```diff
@@ -19,7 +19,8 @@ <h2 class="dot-starter-title">{{ 'starter.title' | dm }}</h2>
                     <a
                         class="dot-starter-top-main__block"
                         data-testid="starter.main.link.data.model"
-                        href="#"
+                        href="javascript:void(0)"
```
Comment (fmontes) [inline src/app/portlets/dot-starter/dot-starter.component.html:22]: To navigate in a `<a>` we should use https://angular.io/api/router/RouterLink

File: src/app/portlets/dot-starter/dot-starter.component.ts [hunk 2]
```diff
@@ -11,7 +12,18 @@ import { DotToolGroupService } from '@services/dot-tool-group/dot-tool-group.ser
 export class DotStarterComponent implements OnInit {
     username: string;
 
-    constructor(private route: ActivatedRoute, private dotToolGroupService: DotToolGroupService) {}
+    portletKeyLink = {
+        'starter.main.link.data.model': '/content-types-angular/create/content',
+        'starter.main.link.add.content': '/c/content/new/webPageContent',
+        'starter.main.link.design.layout': '/templates/new/designer',
+        'starter.main.link.create.page': '/c/content/new/htmlpageasset'
+    };
```
Comment (fmontes) [inline src/app/portlets/dot-starter/dot-starter.component.ts:20]: Directly in the hmt routerLink

File: src/app/portlets/dot-starter/dot-starter.component.html [hunk 3]
```diff
@@ -205,7 +220,12 @@ <h4>{{ 'starter.footer.link.examples.title' | dm }}</h4>
         </a>
     </div>
     <div class="p-lg-4 p-md-12">
-        <a href="#" data-testid="starter.footer.link.community" class="dot-starter-bottom__block">
+        <a
+            href="https://dotcms.com/forum/"
+            data-testid="starter.footer.link.community"
```
Comment (fmontes) [inline src/app/portlets/dot-starter/dot-starter.component.html:225]: Isn't should this be camelCase? we need to doc this thing we keep having this issues.

File: src/app/portlets/dot-starter/dot-starter.component.scss [hunk 4]
```diff
@@ -170,9 +170,9 @@ $number-link-icon-size: 2.28rem;
         box-shadow: $md-shadow-4;
         display: flex;
         flex-grow: 1;
+        min-height: 97px;
```
Comment (fmontes) [inline src/app/portlets/dot-starter/dot-starter.component.scss:173]: px? why?

File: src/app/portlets/dot-starter/dot-starter.component.scss [hunk 5]
```diff
@@ -193,6 +193,7 @@ $number-link-icon-size: 2.28rem;
 
     .dot-starter-bottom__link-data {
         align-self: center;
+        margin-right: 2rem;
```
Comment (fmontes) [inline src/app/portlets/dot-starter/dot-starter.component.scss:196]: spacing?

**Review by fmontes — CHANGES_REQUESTED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**